trade-finance-manager-ui/templates/case/underwriting/pricing-and-risk/probability-of-default.njk

```diff
@@ -5,6 +5,8 @@
 
 {% extends "index.njk" %}
 
+{% block pageTitle %}What’s the probability of default for {{ deal.submissionDetails.supplierName}}?{% endblock %}
```

Comment:
Multiline formatting please.

Reply:
Changed all pageTitle blocks to multiline format.